### PR TITLE
Spp 979 new logger

### PR DESCRIPTION
**Pull request by rhodriguerrier**
Add new spp logger to BMI disclosure wrangler and methods. All tests now passing

### CODE REVIEW

File: disclosure_wrangler.py [hunk 1]
```diff
@@ -145,7 +162,9 @@ def lambda_handler(event, context):
             "explanation": explanation,
             "total_columns": total_columns,
             "unique_identifier": unique_identifier,
-            "run_id": run_id
+            "run_id": run_id,
+            "environment": environment,
+            "survey": survey
```
Comment (Piwington) [inline disclosure_wrangler.py]: Alphabetical please. :)

File: tests/test_disclosure.py [hunk 2]
```diff
@@ -27,7 +27,9 @@
         "publishable_indicator": "publish",
         "total_columns": ["Q608_total", "Q606_other_gravel"],
         "unique_identifier": ["responder_id"],
-        "run_id": "666"
+        "run_id": "666",
+        "environment": "sandbox",
+        "survey": "BMI_SG"
```
Comment (Piwington) [inline tests/test_disclosure.py]: Alphabet

File: tests/test_disclosure.py [hunk 3]
```diff
@@ -42,7 +44,9 @@
         "threshold": "7",
         "total_columns": ["Q608_total", "Q606_other_gravel"],
         "unique_identifier": ["responder_id"],
-        "run_id": "666"
+        "run_id": "666",
+        "environment": "sandbox",
+        "survey": "BMI_SG"
```
Comment (Piwington) [inline tests/test_disclosure.py]: Alphabet

File: tests/test_disclosure.py [hunk 4]
```diff
@@ -59,7 +63,9 @@
         "top2_column": "second_largest_contributor",
         "total_columns": ["Q608_total", "Q606_other_gravel"],
         "unique_identifier": ["responder_id"],
-        "run_id": "666"
+        "run_id": "666",
+        "environment": "sandbox",
+        "survey": "BMI_SG"
```
Comment (Piwington) [inline tests/test_disclosure.py]: Alphabet

**Review by Piwington — CHANGES_REQUESTED** [review]
Good code. Just Alphabetical. :)

**Review by Piwington — APPROVED** [review]

**Review by glanvl — APPROVED** [review]

**Pull request merged**